bundles/org.eclipse.swt/Eclipse SWT Browser/common/org/eclipse/swt/browser/Browser.java

```diff
@@ -1187,6 +1189,17 @@ public boolean setUrl (String url, String postData, String[] headers) {
 	return webBrowser.setUrl (url, postData, headers);
 }
 
+/**
+ * Checks if the location supplied is the location for setting custom text in the browser
```

Comment:
Would be good to extend the documentation a bit on what "setting custom text" means. Somethin like:
```suggestion
 * Checks if the location supplied is the location for setting custom text in the browser. This means, when passing the browser URL to this method after a custom text has been set via {@link #setText(String)} will yield true as long as no further navigation to some other location is performed.
```

Reply:
Done